Sources/SwiftProtobuf/AnyMessageStorage.swift

```diff
@@ -162,6 +162,36 @@ internal class AnyMessageStorage {
     throw AnyUnpackError.malformedAnyField
   }
 
+
```

Comment:
Yes.  I moved the text encode/decode support into an extension below, to match how we had grouped the JSON special support.